casa/OS/HostInfo.h

```diff
@@ -131,12 +131,22 @@ class HostInfo
     static ptrdiff_t swapFree();
     // </group>
 
+    // Introduce run-time memory limit re-specification capabilities
+    // <group>
+    static void setMemoryTotal(int memory);
```

Comment:
swig so far I know does not use xml files, so I guess you mean casas homebrew'd gcwrap. There probably no, but can you even bind casacore functions with that?